docs/source/developers/guide/step_by_step/building.rst

```diff
@@ -35,17 +35,83 @@
 Building Arrow's libraries 🏋🏿‍♀️
```

Comment:
It's not in this PR, but this should probably be "Building the Arrow libraries".